dynamic/README.md

```diff
@@ -272,3 +272,15 @@ Moving the release process into [github.com/pulumi/pulumi-terraform-bridge](http
 
 - [Provide a centralized lookup for PluginDownloadURL #4851 ](https://github.com/pulumi/registry/issues/4851) (with CLI adoption)
 - [Support nested release labels #4852](https://github.com/pulumi/registry/issues/4852)
+
+
+## Debugging dynamically bridged providers
+
+Dynamically bridged providers allow the Terraform provider interactions to be recorded in order to allow maintainers who do not have access to the Terraform provider to debug them. To do that:
```

Comment:
For my 2c, this user experience - what we are asking users to do - is not great. Can we do better? Even if we ship this for now, what would it take to make this work seamlessly with

https://www.pulumi.com/docs/iac/support/troubleshooting/

Reply:
Yeah, I believe we discussed having the GRPC logs in the normal logs and then we can just grep the right lines in it. Opened https://github.com/pulumi/pulumi-terraform-bridge/issues/2535 for this.